great_expectations/experimental/datasources/sql_datasource.py

```diff
@@ -52,69 +54,63 @@ class SQLDatasourceWarning(UserWarning):
     pass
 
 
-@pydantic_dc.dataclass(frozen=True)
-class ColumnSplitter:
+class _ColumnSplitter(ExperimentalBaseModel):
     column_name: str
     method_name: str
 
+    @property
+    def param_names(self) -> List[str]:
+        raise NotImplementedError
+
+    def splitter_method_kwargs(self) -> Dict[str, Any]:
+        raise NotImplementedError
+
+    def request_options_to_batch_spec_kwarg_identifiers(
```

Comment:
@billdirks Would it make sense for us to spell "batch_request" out in names here for an easier interpretation so that "request" is not confused with something else (e.g., Web request)?  Otherwise, there is some risk of cognitive overload and extra time spent searching through documentation.  Thanks.

Reply:
Done.